### PR TITLE
Rebuilt Tor 0.4.7.8 (new Windows signing key)

### PR DESCRIPTION
**Pull request by fmarier**
See brave/tor_build_scripts#103.

### CODE REVIEW

**Review by mihaiplesa — APPROVED** [review]

**Comment by fmarier** [comment]
Needs QA approval before merging.

**Review by mihaiplesa — APPROVED** [review]

**Pull request merged**
